src/static/css/site.css

```
@@ -657,7 +657,7 @@ summary .group-name {
   border-bottom: 1px solid var(--dim-color);
 }
 
-.wiki-search-result:hover::before {
+.wiki-search-result:hover::before, .wiki-search-result:focus::before {
```

We usually keep one CSS selector per line - can you move this new one to its own line?

```suggestion
.wiki-search-result:hover::before,
.wiki-search-result:focus::before {
```